package.json

```diff
@@ -1,21 +1,31 @@
 {
   "name": "travel-and-help",
   "version": "1.0.0",
-  "description": "",
+  "description": "feed the homeless in chelyabinsk",
   "private": true,
+  "babel": {
+    "presets": [
+      "es2015",
+      "stage-0",
+      "react"
+    ]
+  },
   "scripts": {
-    "start:back": "cd src/back && nodemon",
-    "start:front": "npm run build",
+    "prestart": "npm install",
     "start": "npm-run-all start:front start:back",
+    "start:front": "npm run build",
+    "start:back": "cd src/back && nodemon",
+    "watch": "npm-run-all -p watch:*",
     "watch:front": "webpack-dev-server --config webpack/index.js",
     "watch:back": "npm run start:back",
     "watch:db": "mongod --dbpath .local-db",
-    "watch": "npm-run-all -p watch:*",
     "build": "webpack --config webpack/index.js",
+    "build:android": "cordova build android",
```

Comment:
Yes you right. We build the bundle with regular `npm run build` script.
And `npm run build:android` handle it from `www` directory we've got so far into native platform